brainstormforce/ultimate-addons-for-gutenberg
# Fix: Tabs block - whole tab not clickable.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/9e628fb0-9ac4-4f3d-9990-aa7314bd36a5?view=6d3bcc3d-b047-4739-8e05-d9621e9f9445&t=attributes

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop Tabs on the page. Select a different style.
- Click on the tabs button to check is it clickable in the editor as well as on the frontend.
- The layout does not break and everything works properly in both ends.

### Checklist:
